shell/components/ResourceDetail/Masthead.vue

```diff
@@ -636,10 +658,7 @@ export default {
     }
   }
 
-  div.actions-container > div.actions {
-    display: flex;
-    flex-direction: row;
-    justify-content: flex-end;
+  .resource-external {
+    font-size: 18px;
   }
```

Comment:
Isn't this causing issues with actions on detail pages?

Reply:
It's a rebase typo. Fixed